Lib/sre_constants.py

```diff
@@ -21,6 +21,17 @@
 # should this really be here?
 
 class error(Exception):
+    '''Exception raised for invalid regular expressions.
+
+    The error instance has the following additional attributes:
+
+    msg: The unformatted error message
+    pattern: The regular expression pattern
+    pos: The index of pattern where compilation failed
```

Comment:
The wording is copied from the module documentation (see [bpo-22578](https://bugs.python.org/issue22578)). Should it be updated too?

Attributes `pattern`, `pos`, `lineno` and `colno` can be `None` if it is hard to determine the point of failure.